au/io.hh

```diff
@@ -25,7 +25,7 @@ namespace au {
 // Streaming output support for Quantity types.
 template <typename U, typename R>
 std::ostream &operator<<(std::ostream &out, const Quantity<U, R> &q) {
-    out << q.in(U{}) << " " << unit_label(U{});
+    out << +q.in(U{}) << " " << unit_label(U{});
```

Comment:
Nice, elegant change!  I think this is a good use of integer promotion here (although I dislike integer promotion in general as a language feature of C and C++).

Since this is pretty non-obvious, how about a comment here?  Something to the effect of, "Unary plus causes integer promotion.  This helps when the rep is `int8_t`, which could otherwise be interpreted as `char` in a streaming context."

Reply:
Done in 3689593.